lib/rpc/SingleJsonRpcProvider.ts

```diff
@@ -332,7 +336,7 @@ export class SingleJsonRpcProvider extends StaticJsonRpcProvider {
       perf.latencyInMs = Date.now() - perf.startTimestampInMs
       this.checkLastCallPerformance(perf)
       this.updateHealthyStatus()
-      if (this.enableDbSync) {
```

Comment:
You need to log two metrics to indicate the sampled DB Sync is indeed the desired sampling probability. See https://github.com/Uniswap/routing-api/blob/663523f6d7ed1e275e4b1fbec783279cb94246c6/lib/handlers/pools/provider-migration/v3/traffic-switch-v3-pool-provider.ts#L55 as an example.

Reply:
Yes, now use two metrics as you suggested.